cypress/integration/allEnv/chcekout/warehouses.js

```diff
@@ -0,0 +1,67 @@
+import faker from "faker";
+
+import { createCheckout } from "../../../apiRequests/Checkout";
+import { getDefaultChannel } from "../../../utils/channelsUtils";
+import {
+  createProductInChannel,
+  createTypeAttributeAndCategoryForProduct,
+  deleteProductsStartsWith
+} from "../../../utils/products/productsUtils";
+import {
+  createShipping,
+  deleteShippingStartsWith
+} from "../../../utils/shippingUtils";
+
+describe("Warehouses in checkout", () => {
+  const startsWith = `CyWarehouseCheckout`;
+  let defaultChannel;
+  let usAddress;
+  let plAddress;
+  let warehouse;
+  it("should not be possible to buy product for country not listed in warehouse", () => {
```

Comment:
Please add an empty line.

Reply:
Please don't act as a linter. Also why would you want to put an empty line here?

Reply:
Oh okay I get it

Reply:
```suggestion
  
  it("should not be possible to buy product for country not listed in warehouse", () => {
```